__tests__/mosaic512px.mjs

```
@@ -0,0 +1,352 @@
+  dbQueryHandlers.set(name, handler);
+}
+
+jest.unstable_mockModule("../src/db.mjs", () => ({
+  getClient: jest.fn(() => {
+    const query = ({ name, values }) => {
+      if (dbQueryHandlers.has(name)) {
+        const handler = dbQueryHandlers.get(name);
+        return handler(values);
+      }
+
+      throw new Error("undefined database query with name: " + name);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)